src/app/lib/config/services/kyrgyz.js

```diff
@@ -36,15 +36,15 @@ const service = {
     swPath: '/sw.js',
     frontPageTitle: 'Башталгыч бет',
     translations: {
-      seeAll: 'See all',
+      seeAll: 'Баарын көрүү',
       home: 'Башталгыч бет',
-      currentPage: 'Current page',
+      currentPage: 'Ачылып турган баракча',
       skipLinkText: 'Сайтка өтүү',
-      relatedContent: 'Related content',
+      relatedContent: 'Тема боюнча башка макалалар',
       error: {
         404: {
           statusCode: '404',
-          title: '404 - Барак табылган жок',
+          title: 'Баракча табылган жок',
           message:
             'Бул веб-даректи туура эмес тергенден келип чыгышы мүмкүн. Даректи жана анын жазылышын текшерип көрүңүз.',
           solutions: [
```

Comment:
These have translations available now.

Reply:
I have added the errors' solutions translations